lib/src/state_machine.dart

```diff
@@ -137,10 +139,32 @@ class State extends Disposable implements Function {
     return _machine.current == this;
   }
 
+  /// When starting the machine
+  /// by calling stm.start(initialState)
+  /// the initialState State receives an
+  /// onEnter event where the from state
+  /// is this stub one.
+  /// 
+  /// You can check that condition with:
+  /// 
+  /// initialState.onEnter((change) {
+  ///   if (!change.from.isNone) {
+  ///     // do something here
+  ///   }
+  /// })
+  /// 
+  /// to ignore this false initial event.
+  /// 
+  /// bool, true if this state is None,
+  /// false otherwise
+  bool get isNone {
+    return name == _NONE_STATE_NAME;
```

Comment:
Instead of adding `isNone` to the API for every `State` instance, could we add a `static` field to represent the none state and use that? Then the `isInitial` getter below could be:

```dart
bool get isInitial => from == State.none
```

Reply:
what about this?